tests/app/test_jinja_filters.py

```diff
@@ -129,6 +129,16 @@ def test_format_date_month_year(self):
         # Then
         self.assertEqual(format_value, "<span class='date'>January 2017</span>")
 
+    def test_format_date_time(self):
+        # Given
+        date_time = '2018-03-28T13:59:00.659262'
+
+        # When
+        format_value = format_datetime(date_time, date_format='%d %B %Y at %H:%M')
```

Comment:
What happens on 28th October when BST ends?

Reply:
Clocks go forward 🙄

Reply:
and the evaluation of `if datetime.now().hour > datetime.utcnow().hour:` changes

Reply:
@hamishtaplin also, they go back ;)

Reply:
Not in my house.

Reply:
@jonnyshaw89 Yeah I know, that's what I wrote and I didn't just edit it.